packages/wallet/backend/src/gatehub/client.ts

```diff
@@ -348,14 +412,19 @@ export class GateHubClient {
     method: HTTP_METHODS,
     url: string,
     body?: string,
-    managedUserUuid?: string
+    managedUserUuid?: string,
+    token?: string
   ) {
     return {
       'Content-Type': 'application/json',
       'x-gatehub-app-id': this.env.GATEHUB_ACCESS_KEY,
       'x-gatehub-timestamp': timestamp,
       'x-gatehub-signature': this.getSignature(timestamp, method, url, body),
-      ...(managedUserUuid && { 'x-gatehub-managed-user-uuid': managedUserUuid })
+      'x-gatehub-card-app-id': this.env.GATEHUB_CARD_APP_ID,
```

Comment:
should this be included in each request?

Reply:
You're right, we only need this for customer and cards related requests.

Reply:
missed to remove `x-gatehub-card-app-id` here